src/core/linter-rules/no-unused-vars.js

```diff
@@ -0,0 +1,66 @@
+// @ts-check
+/**
+ * Linter rule "no-unused-vars".
+ *
+ * Checks that an variable is used if declared (the first use is treated as
+ * declaration).
+ */
+import LinterRule from "../LinterRule.js";
+import { lang as defaultLang } from "../l10n.js";
+import { norm } from "../utils.js";
+
+const name = "no-unused-vars";
+
+const meta = {
```

Comment:
Make sure the message here looks like one you would get from ESLint... that is, it should be clear that a "variable X was defined but never used".

Reply:
We won't have access to "X" in linter message, so maybe:
"Variable was defined, but never used. Add a `data-ignore-unused` attribute to the `<var>`, or consider using `<em>`/ `<code>`"

Reply:
>  "Add a data-ignore-unused attribute to the <var>, or consider using <em>/ <code>"

Only the "data-ignore-unused" seems like an good suggestion... the other not so much as they are working around the problem, not addressing the problem explicitly.

Reply:
No idea what's going on with my comment's formatting above... 😂

Reply:
> Only the "data-ignore-unused" seems like an good suggestion... the other not so much as they are working around the problem, not addressing the problem explicitly.

Maybe author used `<var>` for its style when they meant `<em>` or `<code>` (See https://github.com/w3c/webdriver/pull/1515/files)?

> No idea what's going on with my comment's formatting above... :joy:

Missed backticks on `<em>` :balloon:

Reply:
Also, we can't add a `data-ignore-unused` to the var micro-syntax.

Reply:
Using `<var>` for style is an author error, so let’s not worry about it (or encourage that).